bench/single_file_bench.exs

```diff
@@ -1,27 +0,0 @@
-defmodule Dogma.SingleSourceBench do
```

Comment:
and actually @lpil  while I'm thinking about this how do you feel about having the benchmarking hooked up to travis ci job? It'll stop it getting broken again. Plus if we wanted to we could write something to compare each build to see how the performance has changed over time.